_src/_assets/js/dashboard-map.js

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable */
+
+;(async function loadMap() {
+  const formatter = new Intl.NumberFormat('en-us', {
+    maximumFractionDigits: 1,
+  })
+
+  const colorLimits = [1, 5, 10, 25, 50, 100, 250, 500]
+
+  const getColor = d => {
+    return d > colorLimits[7]
+      ? '#800026'
+      : d > colorLimits[6]
+      ? '#BD0026'
+      : d > colorLimits[5]
+      ? '#E31A1C'
+      : d > colorLimits[4]
+      ? '#FC4E2A'
+      : d > colorLimits[3]
+      ? '#FD8D3C'
+      : d > colorLimits[2]
+      ? '#FEB24C'
+      : d > colorLimits[1]
+      ? '#FED976'
+      : '#FFEDA0'
+  }
+  const joinDataToGeoJson = (geoJSON, stateArray) => {
+    //in addition to joining latest state data to their shapes this also calculates positive cases per million people
+    const stateObject = Object.assign(
+      {},
+      ...stateArray.map(state => ({ [state.state]: state })),
+    )
+    const joinedFeatures = geoJSON.features.map(feature => {
+      const positive = stateObject[feature.properties.STUSPS].positive
+      const population = feature.properties.population
+      const casesPerMil = positive / (population / 1000000)
+      return {
+        ...feature,
+        // geometry: turf.centroid(feature).geometry,// uncomment to use centroid of each state instead of shape
+        properties: {
+          casesPerMil,
+          ...feature.properties,
+          ...stateObject[feature.properties.STUSPS],
+        },
+      }
+    })
+    return { ...geoJSON, features: joinedFeatures }
+  }
+  const initializeMap = () => {
+    const map = L.map('state-map').setView([38.617379, -101.318915], 3)
+    L.tileLayer(
+      'https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token={accessToken}',
+      {
+        attribution:
+          'Map data &copy; <a href="https://www.openstreetmap.org/">OpenStreetMap</a> contributors, <a href="https://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="https://www.mapbox.com/">Mapbox</a>',
+        maxZoom: 18,
+        id: 'mapbox/light-v10',
+        tileSize: 512,
+        zoomOffset: -1,
+        accessToken:
+          'pk.eyJ1IjoiZ29sZWFyeSIsImEiOiJjaXJmNWh5YmgwMDd6ZzNuZXVsOHplYXRmIn0.41N9r7fWdPMGEz60wv5eZw',
```

Comment:
My guess is that this shouldn't be public?

Reply:
it's a public token

Reply:
My bad then -- sorry!